openapi-issuer.yaml

```
@@ -327,14 +357,33 @@ components:
         - productId
         - fiscalCode
 
+    PutSignatureStatusBody:
+      type: object
+      properties:
+        status:
+          $ref: "#/components/schemas/StatusSignatureRequest"
+      required:
+        - status
+
+    StatusSignatureRequest:
+      type: string
+      enum:
```

```suggestion
      x-extensible-enum:
```

see https://opensource.zalando.com/restful-api-guidelines/#112